04_numeronym/madhavansu/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprint(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+func numeronyms(vals ...string) []string {
+	var strcopy []string
+	for _, val := range vals {
+		length := len(val)
+		if length > 3 {
+			first, last := val[0:1], val[length-1:]
```

Comment:
you can directly access first and last values with `val[0]` and `val[length-1]`

Reply:
This is another approach to access the character and I think we are more used to do with val[0] so tried something different.

Reply:
I see this has been changed. Always prefer single access over slice access when just accessing one element at a known index.